pkg/apis/cr/v1alpha1/types.go

```diff
@@ -196,27 +211,42 @@ var _ runtime.Object = (*Blueprint)(nil)
 type Blueprint struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata"`
-	Actions           map[string]*BlueprintAction `json:"actions,omitempty"`
+	// Actions is the list of actions constructing the Blueprint.
+	Actions map[string]*BlueprintAction `json:"actions,omitempty"`
 }
 
 // BlueprintAction describes the set of phases that constitute an action.
 type BlueprintAction struct {
-	Name               string              `json:"name"`
-	Kind               string              `json:"kind"`
-	ConfigMapNames     []string            `json:"configMapNames,omitempty"`
-	SecretNames        []string            `json:"secretNames,omitempty"`
-	InputArtifactNames []string            `json:"inputArtifactNames,omitempty"`
-	OutputArtifacts    map[string]Artifact `json:"outputArtifacts,omitempty"`
-	Phases             []BlueprintPhase    `json:"phases,omitempty"`
-	DeferPhase         *BlueprintPhase     `json:"deferPhase,omitempty"`
+	// Name of the action.
+	Name string `json:"name"`
+	// Kind field is now deprecated.
```

Comment:
```suggestion
	// Kind contains the resource this actions is to be performed on. Kind field is now deprecated.
```

Reply:
worded it like `Kind contains the resource on which this action has to be performed. Kind field is now deprecated.`